### ISSUE TITLE
Add .gitignore for Eclipse

### ISSUE THREAD

**Issue by erickzhao** [erickzhao/Food-Truck-Management-System#3]
Eclipse has a lot of metadata that is unnecessary to track and clutters up the repo. GitHub has an official [.gitignore template](https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore) that we could use.

In order to untrack existing metadata files, we can use `git update-index --assume-unchanged [filepath]`.

This should be done after #1 is completed and is low priority.

**Comment by erickzhao** [comment]
.gitignore is annoying. I'll just close this one.
